doc/release-notes-1.2.0.rst

```diff
@@ -0,0 +1,502 @@
+.. _ncs_release_notes_120:
+
+|NCS| v1.2.0 Release Notes
+##########################
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products.
+It includes the MCUboot and the Zephyr RTOS open source projects, which are continuously integrated and re-distributed with the SDK.
+
+|NCS| v1.2.0 supports product development with the nRF9160 Cellular IoT device.
+It contains reference applications, sample source code, and libraries for Bluetooth Low Energy devices in the nRF52 Series, though product development on these devices is not currently supported with the |NCS|.
+??? What do we say about nRF5340?
```

Comment:
Replace with the following text. This will also include nRF53 in the scope: 
"It contains reference applications, sample source code, and libraries for developing low power wireless applications with nRF52 & nRF53 series devices, though support for these devices is incomplete and not recommended for production."